src/test/java/kitchenpos/ui/TableGroupRestControllerTest.java

```diff
@@ -0,0 +1,51 @@
+package kitchenpos.ui;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.header;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import kitchenpos.ControllerTest;
+import kitchenpos.application.TableGroupService;
+import kitchenpos.domain.TableGroup;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.ResultActions;
+
+@WebMvcTest(TableGroupRestController.class)
+class TableGroupRestControllerTest extends ControllerTest {
+
+    @MockBean
+    private TableGroupService tableGroupService;
+
+    @Test
+    void table_group을_생성할_수_있다() throws Exception {
+        // given
+        TableGroup tableGroup = new TableGroup();
+        tableGroup.setId(1L);
+        when(tableGroupService.create(any(TableGroup.class))).thenReturn(tableGroup);
+        String url = "/api/table-groups";
+
+        // when
+        ResultActions response = postRequestWithJson(url, tableGroup);
+
+        // then
+        response.andExpect(status().isCreated())
+                .andExpect(header().string("location", url + "/" + tableGroup.getId()))
+                .andExpect(content().string(objectMapper.writeValueAsString(tableGroup)));
+    }
+
+    @Test
+    void table_group을_취소할_수_있다() throws Exception {
+        // given
+        String url = "/api/table-groups/1";
+
+        // when
+        ResultActions response = deleteRequest(url);
+
+        // then
+        response.andExpect(status().isNoContent());
+    }
```

Comment:
given, when, then을 주석으로 명시하는 작업이 다소 강박적이라는 느낌이 들어요.
무조건 주석으로 명시하는 컨벤션의 효용에 대해 어떻게 생각하는지 궁금해요. 

```suggestion
    @Test
    void table_group을_취소할_수_있다() throws Exception {
        ResultActions response = deleteRequest("/api/table-groups/1");
        response.andExpect(status().isNoContent());
    }
```